base/linalg/dense.jl

```diff
@@ -5,6 +5,7 @@
 ## BLAS cutoff threshold constants
 
 const SCAL_CUTOFF = 2048
+const TRIA_CUTOFF = 2048
```

Comment:
This number was picked arbitrarily around what seemed to be the cross-over on my machine. I don't know what it should actually be in general.